src/timeline/Timeline.tsx

```diff
@@ -100,24 +108,25 @@ const Timeline = (props: TimelineProps) => {
   const {scrollEvents} = useTimelineOffset({onChangeOffset, scrollOffset, scrollViewRef: scrollView});
 
   const packedEvents = useMemo(() => {
-    const width = dimensionWidth - HOURS_SIDEBAR_WIDTH;
+    const width = constants.screenWidth - HOURS_SIDEBAR_WIDTH;
     return populateEvents(events, width, start);
   }, [events, start]);
 
   useEffect(() => {
+    let initialPosition = 0;
     if (scrollToFirst) {
-      const firstTop = min(map(packedEvents, 'top')) ?? 0;
-      const initPosition = firstTop - calendarHeight.current / (end - start);
-      const verifiedInitPosition = initPosition < 0 ? 0 : initPosition;
```

Comment:
What if initPosition is less than 0?

Reply:
You can see in line 126, that I maxing it to `0` in case we get a negative value
```
y: Math.max(0, initialPosition - HOUR_BLOCK_HEIGHT),
```

Reply:
Yes, but you're maxing it after you reduce the HOUR_BLOCK_HEIGHT... but ok, you're not allowing a negative offset at the top to it doesn't matter